src/v/cluster/service.cc

```diff
@@ -364,12 +364,25 @@ service::hello(hello_request&& req, rpc::streaming_context&) {
 
 ss::future<config_status_reply>
 service::config_status(config_status_request&& req, rpc::streaming_context&) {
-    if (!_config_manager.local().needs_update(req.status)) {
+    // Move to stack to avoid referring to reference argument after a
+    // scheduling point
+    auto status = std::move(req.status);
```

Comment:
ohhh, nice catch.

prefer taking `req` by value. in this case the caller would have had to std::move/make-temporary anyway, and the parameter values are kept alive for the lifetime of the coroutine.